pegasus/router.rb

```diff
@@ -408,6 +408,7 @@ def render_template(path, locals={})
       end
       raise e
     rescue => e
+      Honeybadger.context({path: path, e: e})
```

Comment:
Like we do here?  https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/router.rb#L30